README.md

```diff
@@ -13,8 +13,8 @@ You can download and use WinUI packages in your app using the NuGet package mana
 
 | NuGet Package | Build Status | Latest Versions |
 | --- | --- | --- |
-| [Microsoft.UI.Xaml](https://www.nuget.org/packages/Microsoft.UI.Xaml) <br /> Controls and Fluent Design for UWP apps | [![Build Status](https://dev.azure.com/ms/microsoft-ui-xaml/_apis/build/status/WinUI-Public-MUX-CI?branchName=master)](https://dev.azure.com/ms/microsoft-ui-xaml/_build/latest?definitionId=20?branchName=master) | ![latest stable version](https://img.shields.io/nuget/v/Microsoft.UI.Xaml.svg) <br /> ![latest prerelease version](https://img.shields.io/nuget/vpre/Microsoft.UI.Xaml.svg) |
-| [Microsoft.UI.Xaml.Core.Direct](https://www.nuget.org/packages/Microsoft.UI.Xaml.Core.Direct) <br /> Low-level APIs for middleware components | | ![latest prerelease version](https://img.shields.io/nuget/vpre/Microsoft.UI.Xaml.Core.Direct.svg) |
+| [Microsoft.UI.Xaml](https://www.nuget.org/packages/Microsoft.UI.Xaml) <br /> Controls and Fluent Design for UWP apps | [![Build Status](https://dev.azure.com/ms/microsoft-ui-xaml/_apis/build/status/WinUI-Public-MUX-CI?branchName=master)](https://dev.azure.com/ms/microsoft-ui-xaml/_build/latest?definitionId=20?branchName=master) | [![latest stable version](https://img.shields.io/nuget/v/Microsoft.UI.Xaml.svg)](https://www.nuget.org/packages/Microsoft.UI.Xaml) <br /> [![latest prerelease version](https://img.shields.io/nuget/vpre/Microsoft.UI.Xaml.svg)](https://www.nuget.org/packages/Microsoft.UI.Xaml/2.1.181217001-prerelease) |
```

Comment:
Hmmm ... we're going to forget to update the link whenever the prerelease changes. The image shows the versions, maybe we just have both link out to the base nuget page (https://www.nuget.org/packages/Microsoft.UI.Xaml)?

Reply:
Actually I poked around and divined from [this GitHub issue](https://github.com/NuGet/NuGetGallery/issues/2567) that there is a direct link you can make to latest prerelease. Let's use that.
https://www.nuget.org/packages/Microsoft.UI.Xaml/absoluteLatest

Reply:
Updated.  @kmahone unfortunately I haven't found a way to do that that still gives the desired formatting in the output.